macros/calculate_snapshot_diff_for_day.sql

```diff
@@ -34,7 +34,7 @@ WITH source_ranked AS (
 source_deduped AS (
     SELECT * EXCEPT (row_num),
     -- Add two columns to normalize source and target datasets
-           {{ updated_at_col_name }} AS {{ valid_from_col_name }},
+           CAST({{ updated_at_col_name }} AS TIMESTAMP) AS {{ valid_from_col_name }},
```

Comment:
what are the implications for updating this column data type for existing snapshot tables?